src/Activity.js

```diff
@@ -31,19 +31,20 @@ class Activity extends UserFitness {
     }, 0) / 7)
   }
 
-  returnAverageStepsForWeek(week) {
+returnAverageForWeek(week, activityData) {
     let weekOfData = this.returnWeekOfData(week, this.userData);
-    return Math.floor(weekOfData.reduce((totalSteps, eachDay) => {
-      totalSteps += eachDay.numSteps
-      return totalSteps
+    return Math.floor(weekOfData.reduce((acc, day) => {
+      console.log(day[activityData])
+      acc += day[activityData]
+      return acc
     }, 0) / 7)
   }
-
-  returnAverageStairsForWeek(week) {
+  
+  returnAverageActivityForWeek(week, total) {
```

Comment:
It looks like this function could be refactored into the one above rather than having two there could be one